beets/importer.py

```diff
@@ -1268,11 +1336,11 @@ def tasks(self):
         # for archive imports, send the archive task instead (to remove
         # the extracted directory).
         if self.is_archive:
-            yield archive_task
+            yield archive_task  # type: ignore
```

Comment:
If we _must_ have this in place, it needs a specific error type being ignored. Is it actually needed?

Reply:
You are right not needed anymore